### ISSUE TITLE
sensenet.ru giving 500 error

### ISSUE THREAD

**Issue by jmez11** [mrworf/photoframe#215]
In attempting to set up my frame again (sd card died) i get to the authorization part and see: 

This page isn’t working 
photoframe.sensenet.nu is currently unable to handle this request.
HTTP ERROR 500

I've disabled ad blockers and tried other browsers.  My previously set up and working credential json was what i used.  I tried resetting up the credentials from scratch, just to be sure and those failed too.

Any help would be appreciated!

**Comment by mrworf** [comment]
Odd, I just tried calling https://photoframe.sensenet.nu/photoframe/?register=10.0.0.1:80 from my mobile device (without wifi) to make sure I get the same flow and it works (returns a hash). Can you try the link above and see what you get? Also, just https://photoframe.sensenet.nu should redirect to this github project.

**Comment by jmez11** [comment]
That doesn't look like the url that my authorize button uses.  I see the URL like: 

https://photoframe.sensenet.nu/?state=xxx&code=xxx&scope=https://www.googleapis.com/auth/photoslibrary.readonly

**Comment by mrworf** [comment]
That's the correct URL, the browser calls my link first to get the state value (the hash value). See https://github.com/mrworf/photoframe#how-come-you-contact-photoframesensenetnu- for details on how this flow works.

Reason I posted the link was to make sure you could access the site to get the hash in the first place.

**Comment by jmez11** [comment]
Yeah, no problems with the first url.  The 500 error happens with the redirect url.   I'm able to work around it by manually redirecting myself, so ongoing access is not an issue.  It's only when setting up the new source.  Can you try the full flow by authorizing a new google photos source yourself?   It's still failing when i try adding and authorizing.
